portfolio/src/main/webapp/index.html

```diff
@@ -2,11 +2,23 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Jess DeJong - STEP Intern</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+    <div class="header">
+      <img src="images/Profile_photo_Jess.jpg" alt="Profile photo of Jess" height="250">
+      <h1>Jess's Portfolio</h1>
+      <strong>
+        <a href="#">About Me</a>
+        <a href="#">Projects</a>
+        <a href="#">Experience</a>
+        <a href="#">Hobbies</a>
+        <a href="#">Socials & Contact</a>
+      </strong>
```

Comment:
nit: consider using a styled list for this, e.g.

```html
<ul class="navigation">
  <li><a href="#">About Me</a>
  <!-- ... -->
</ul>
```

```css
.navigation li {
  list-style: none;
  display: inline-block;
}
```

Reply:
This looks nicer, I'll look into it!